executor/common.h

```diff
@@ -467,8 +469,13 @@ static uintptr_t syz_extract_tcp_res(uintptr_t a0, uintptr_t a1, uintptr_t a2)
 	}
 
 	struct tcp_resources* res = (struct tcp_resources*)a0;
+#if defined(SYZ_EXECUTOR) || defined(SYZ_HANDLE_SEGV)
```

Comment:
This is now 7 lines of code instead of 1 and 4 preprocessor invocations instead of 0. This is manually written code that we modify/review/read a lot. It should look like `NONFAILING(do_access());` or along these lines. And then some magic to do the right thing during repro generation.
Can preprocessor do this if we give it some additional flags? If not, then just run a regexp that replaces "NONFAILING(.*);\n" with ".*;\n" if we don't have other ideas. If we run it on the whole output, then as a bonus we also don't need to deal with it in generated code (always generate NONFAILING).